Taken from tcpdump-3.8.2-14.FC4.src.rpm with some similar fixes for arpwatch.8
and and fixes unescaped hyphen in arpwatch and arpsnmp man pages.

diff -Naru arpwatch-2.1a15.orig/arpsnmp.8 arpwatch-2.1a15/arpsnmp.8
--- arpwatch-2.1a15.orig/arpsnmp.8	2000-09-18 00:34:48.000000000 +0400
+++ arpwatch-2.1a15/arpsnmp.8	2006-09-22 19:21:55.000000000 +0400
@@ -22,7 +22,7 @@
 .TH ARPSNMP 8 "17 September 2000"
 .UC 4
 .SH NAME
-arpsnmp - keep track of ethernet/ip address pairings
+arpsnmp \- keep track of ethernet/ip address pairings
 .SH SYNOPSIS
 .B arpsnmp
 [
@@ -41,7 +41,7 @@
 and reports certain changes via email.
 .B Arpsnmp
 reads information from a file (usually generated by
-.BR snmpwalk (8)).
+.BR snmpwalk (1)).
 .LP
 The
 .B -d
@@ -62,9 +62,9 @@
 .LP
 .SH "REPORT MESSAGES"
 (See the
-.BR arpwatch (1)
+.BR arpwatch (8)
 man page for details on the report messages generated by
-.BR arpsnmp (1).)
+.BR arpsnmp (8).)
 .SH FILES
 .na
 .nh
@@ -79,7 +79,7 @@
 .na
 .nh
 .BR arpwatch (8),
-.BR snmpwalk (8),
+.BR snmpwalk (1),
 .BR arp (8)
 .ad
 .hy
diff -Naru arpwatch-2.1a15.orig/arpwatch.8 arpwatch-2.1a15/arpwatch.8
--- arpwatch-2.1a15.orig/arpwatch.8	2000-10-09 00:31:28.000000000 +0400
+++ arpwatch-2.1a15/arpwatch.8	2006-09-22 19:22:07.000000000 +0400
@@ -22,7 +22,7 @@
 .TH ARPWATCH 8 "8 October 2000"
 .UC 4
 .SH NAME
-arpwatch - keep track of ethernet/ip address pairings
+arpwatch \- keep track of ethernet/ip address pairings
 .SH SYNOPSIS
 .na
 .B arpwatch
@@ -101,9 +101,9 @@
 .LP
 .SH "REPORT MESSAGES"
 Here's a quick list of the report messages generated by
-.BR arpwatch (1)
+.BR arpwatch (8)
 (and
-.BR arpsnmp (1)):
+.BR arpsnmp (8)):
 .TP
 .B "new activity"
 This ethernet/ip address pair has been used for the first time six
